tiatoolbox/__init__.py

```diff
@@ -46,7 +50,15 @@
 
 
 class DuplicateFilter(logging.Filter):
-    def filter(self, record):
+    """Define an object to filter duplicate logs.
+
+    The DuplicateFilter filters logs to avoid printing them multiple times
+    while running code in a loop.
+
+    """
+
+    def filter(self, record):  # noqa: A003
+        """Filters input record."""
```

Comment:
```suggestion
        """Filter records."""
```

Reply:
Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/625/commits/bb24fccd10322177197c25b2192531eb94e9a6b4